src/app/artifact/artifact.component.ts

```diff
@@ -92,7 +92,7 @@ export class ArtifactComponent implements OnInit {
   }
 
   mavenCentralBadge(g: string, a: string, v: string): string {
-    return `[![Maven Central](https://img.shields.io/maven-central/v/${g}/${a}.svg?label=Maven%20Central)](http://search.maven.org/#search%7Cga%7C1%7Cg%3A%22${g}%22%20a%3A%22${a}%22)`
+    return `[![Maven Central](https://img.shields.io/maven-central/v/${g}/${a}.svg?label=Maven%20Central)](https://search.maven.org/search?q=g:%22${g}%22%20AND%20a:%22${a}%22&core=gav)`
```

Comment:
This looks good, though I don't believe in the classic we added `&core=gav`. Without it it will produce the same results as in classic, e.g. 

- http://search.maven.org/classic/#search%7Cga%7C1%7Cg%3A%22junit%22%20a%3A%22junit%22

Will have the same results as 

- https://search.maven.org/search?q=g:%22junit%22%20AND%20a:%22junit%22

Where as adding `core=gav` will add a total of 24 results
- https://search.maven.org/search?q=g:%22junit%22%20AND%20a:%22junit%22&core=gav

Reply:
Yes, you're  right, I will perform a minor change and remove the core=gav.